FordLabs/retroquest
# Update eslint/prettier settings & run prettier and eslint fix

## Overview
Updated main with eslint/prettier config updates, ran `eslint --fix` and `prettier --fix`, and manually resolved any errors/warnings that weren't auto fixed.

These changes were made in the branch `react-migration`. Moving to main to avoid rebase issues.

Connects N/A

### Demo
No functionality should have changed.

## Testing Instructions
* Ensure all unit tests pass
* Ensure all cypress tests pass
* Run `yarn build` and ensure the app builds properly
* Run `yarn eslint` and ensure no warnings or errors show up
* Run `yarn prettier` and ensure no warnings or errors show up
* Ensure app works the same as it did before
